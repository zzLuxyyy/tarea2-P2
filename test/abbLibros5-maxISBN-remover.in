crearABBLibrosVacio
crearFecha 01/05/1992
crearLibro 140003446 "Jazz" "Toni" "Morrison" "Una historia de amor y crimen en el Harlem de los años 20" 3
insertarLibroABBLibros
maxISBNLibroABBLibros
crearFecha 16/07/1951
crearLibro 316769177 "El guardián en el centeno" "J.D." "Salinger" "Holden narra sus días tras ser expulsado del colegio, en busca de sentido" 1
insertarLibroABBLibros
maxISBNLibroABBLibros
crearFecha 01/01/1971
crearLibro 204578912 "Las venas abiertas de América Latina" "Eduardo" "Galeano" "Un ensayo crítico sobre la historia de la explotación en América Latina" 7
insertarLibroABBLibros
maxISBNLibroABBLibros
crearFecha 01/01/2008
crearLibro 880619315 "La soledad de los números primos" "Paolo" "Giordano" "Dos almas heridas incapaces de coincidir en el amor" 1
insertarLibroABBLibros
maxISBNLibroABBLibros
removerLibroABBLibros 316769177
maxISBNLibroABBLibros
removerLibroABBLibros 880619315
maxISBNLibroABBLibros
crearFecha 01/01/2008
crearLibro 880619315 "La soledad de los números primos" "Paolo" "Giordano" "Dos almas heridas incapaces de coincidir en el amor" 1
insertarLibroABBLibros
crearFecha 16/07/1951
crearLibro 316769177 "El guardián en el centeno" "J.D." "Salinger" "Holden narra sus días tras ser expulsado del colegio, en busca de sentido" 1
insertarLibroABBLibros
crearFecha 01/10/1995
crearLibro 843397030 "Ensayo sobre la ceguera" "José" "Saramago" "Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano" 2
insertarLibroABBLibros
maxISBNLibroABBLibros
crearFecha 01/09/2005
crearLibro 999001122 "Nunca me abandones" "Kazuo" "Ishiguro" "Jóvenes clonados enfrentan su destino en un mundo distópico" 2
insertarLibroABBLibros
imprimirABBLibros
maxISBNLibroABBLibros
removerLibroABBLibros 880619315
imprimirABBLibros
removerLibroABBLibros 843397030
imprimirABBLibros
liberarABBLibros
Fin
