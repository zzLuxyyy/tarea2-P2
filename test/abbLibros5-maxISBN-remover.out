1>Se ejecutó crearTABBLibrosVacio exitosamente.
2>3>4>Se insertó el libro con ISBN 140003446 al árbol.
5>El mayor ISBN en el árbol es 140003446.
6>7>8>Se insertó el libro con ISBN 316769177 al árbol.
9>El mayor ISBN en el árbol es 316769177.
10>11>12>Se insertó el libro con ISBN 204578912 al árbol.
13>El mayor ISBN en el árbol es 316769177.
14>15>16>Se insertó el libro con ISBN 880619315 al árbol.
17>El mayor ISBN en el árbol es 880619315.
18>El libro con ISBN 316769177 se removió del árbol.
19>El mayor ISBN en el árbol es 880619315.
20>El libro con ISBN 880619315 se removió del árbol.
21>El mayor ISBN en el árbol es 204578912.
22>23>24>Se insertó el libro con ISBN 880619315 al árbol.
25>26>27>Se insertó el libro con ISBN 316769177 al árbol.
28>29>30>Se insertó el libro con ISBN 843397030 al árbol.
31>El mayor ISBN en el árbol es 880619315.
32>33>34>Se insertó el libro con ISBN 999001122 al árbol.
35>Libro 140003446
Título: Jazz
Autor: Toni Morrison
Descripción: Una historia de amor y crimen en el Harlem de los años 20
Género: 3
Fecha de edición: 1/5/1992
Libro 204578912
Título: Las venas abiertas de América Latina
Autor: Eduardo Galeano
Descripción: Un ensayo crítico sobre la historia de la explotación en América Latina
Género: 7
Fecha de edición: 1/1/1971
Libro 316769177
Título: El guardián en el centeno
Autor: J.D. Salinger
Descripción: Holden narra sus días tras ser expulsado del colegio, en busca de sentido
Género: 1
Fecha de edición: 16/7/1951
Libro 843397030
Título: Ensayo sobre la ceguera
Autor: José Saramago
Descripción: Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano
Género: 2
Fecha de edición: 1/10/1995
Libro 880619315
Título: La soledad de los números primos
Autor: Paolo Giordano
Descripción: Dos almas heridas incapaces de coincidir en el amor
Género: 1
Fecha de edición: 1/1/2008
Libro 999001122
Título: Nunca me abandones
Autor: Kazuo Ishiguro
Descripción: Jóvenes clonados enfrentan su destino en un mundo distópico
Género: 2
Fecha de edición: 1/9/2005
36>El mayor ISBN en el árbol es 999001122.
37>El libro con ISBN 880619315 se removió del árbol.
38>Libro 140003446
Título: Jazz
Autor: Toni Morrison
Descripción: Una historia de amor y crimen en el Harlem de los años 20
Género: 3
Fecha de edición: 1/5/1992
Libro 204578912
Título: Las venas abiertas de América Latina
Autor: Eduardo Galeano
Descripción: Un ensayo crítico sobre la historia de la explotación en América Latina
Género: 7
Fecha de edición: 1/1/1971
Libro 316769177
Título: El guardián en el centeno
Autor: J.D. Salinger
Descripción: Holden narra sus días tras ser expulsado del colegio, en busca de sentido
Género: 1
Fecha de edición: 16/7/1951
Libro 843397030
Título: Ensayo sobre la ceguera
Autor: José Saramago
Descripción: Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano
Género: 2
Fecha de edición: 1/10/1995
Libro 999001122
Título: Nunca me abandones
Autor: Kazuo Ishiguro
Descripción: Jóvenes clonados enfrentan su destino en un mundo distópico
Género: 2
Fecha de edición: 1/9/2005
39>El libro con ISBN 843397030 se removió del árbol.
40>Libro 140003446
Título: Jazz
Autor: Toni Morrison
Descripción: Una historia de amor y crimen en el Harlem de los años 20
Género: 3
Fecha de edición: 1/5/1992
Libro 204578912
Título: Las venas abiertas de América Latina
Autor: Eduardo Galeano
Descripción: Un ensayo crítico sobre la historia de la explotación en América Latina
Género: 7
Fecha de edición: 1/1/1971
Libro 316769177
Título: El guardián en el centeno
Autor: J.D. Salinger
Descripción: Holden narra sus días tras ser expulsado del colegio, en busca de sentido
Género: 1
Fecha de edición: 16/7/1951
Libro 999001122
Título: Nunca me abandones
Autor: Kazuo Ishiguro
Descripción: Jóvenes clonados enfrentan su destino en un mundo distópico
Género: 2
Fecha de edición: 1/9/2005
41>42>Fin.
